_posts/2016/2016-02-16-pipeline-elk-test-evaluation-2.md

```diff
@@ -0,0 +1,290 @@
+---
+layout: post
+title: "Automated test evaluation with ELK in the pipeline: Implemented solution"
+date: "2016-02-16 06:16:02"
+icon: wrench
+categories: tech-stories
+authors: ["Benjamin N.", "Bastian K."]
+---
+
+We implemented a Selenium test report database with Elasticsearch, Logstash, Docker, CircleCI and Jenkins to ease the test evaluation process in our Continuous Delivery Pipeline. Last week we have already introduced you to the [background of the project](https://developer.epages.com/blog/2016/02/11/pipeline-elk-test-evaluation-1.html) and today we want to get your hands on the actual development steps. Hence, this post showcases the various parts of the implemented solution and discusses the pragmatic benefits for our pipeline and our speed-up for massive regression test evaluation.
+
+Furthermore, this article should serve as an outline of the consolidated technical expertise gained throughout the engineering process of this project.
+
+## Implemented Solution
```

Comment:
I will adapt the headline capitalization to fit our policy :+1: 

Some popular newspapers/websites (e.g. NYT, WSJ) capitalize nearly all words except some filling words ("the,to,in"). So it was a tough choice. Should have look it up in your space first. :-)